### DISCUSSION TITLE
Common file access control require admin role?

### DISCUSSION THREAD

**Discussion by MorganReid** [ONLYOFFICE/DocumentServer#2371] [Q&A] [answered]
I self-deployed an office-related service and everything worked fine. But I have a small problem.
I logged in with an account with the `admin` role, created a new file `test0904` under `common`, and then selected `change owner `to set the owner of `test0904` to `admin ldap` account.
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/52235796/eeafe81b-5ae3-4a24-b36f-3a5b1968ba2c)

I use the `admin ldap` account to log in, and I see the file `test0904` under common. I want to set the permissions for this file, but I find that there is no such option, but the account of the `admin` role does.
![image](https://github.com/ONLYOFFICE/DocumentServer/assets/52235796/67eccc8e-b47e-4036-aaec-1173241f7dfa)

Does `Access (Permission setting)` require that the account be in the `admin role`,Even if it is the owner account of the file? In fact, I think that since the file owner has been assigned to an account, the account should have permission control of the file, otherwise  still need an account with the admin role to operate.

**Comment by trofim24** [comment, marked as answer]
I created [issue](https://github.com/ONLYOFFICE/CommunityServer/issues/481) and moved it to Community Server
